uploadNeuroDB/minc_insertion.pl

```diff
@@ -145,44 +145,45 @@
 &Getopt::Tabular::SetHelp($Help, $Usage);
 &Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit 1;
```

Comment:
@nicolasbrossard I just realize I missed one exit code here. 
`exit 1` should actually be `exit $NeuroDB::ExitCodes::GETOPT_FAILURE`

Do you mind making the change at the same time as you PR?

Reply:
Done.